README.md

```diff
@@ -459,10 +459,56 @@ pairwise constructs as found in e.g. `let` and `cond`.
                                      LinkedBlockingQueue]))
     ```
 
+* <a name="10-loc-per-fn-limit"></a>
+  Avoid functions longer than 10 LOC (lines of code). Ideally, most
+  functions will be shorter than 5 LOC.
+<sup>[[link](#10-loc-per-fn-limit)]</sup>
+
+* <a name="4-positional-fn-params-limit"></a>
+  Avoid parameter lists with more than three or four positional parameters.
+<sup>[[link](#4-positional-fn-params-limit)]</sup>
+
+* <a name="forward-references"></a>
+  Avoid forward references.  They are occasionally necessary, but such occasions
+  are rare in practice.
+<sup>[[link](#forward-references)]</sup>
+
+## Namespaces
+
+* <a name="stuart-sierra-namespace-guideline"></a>
+  Adopt [Stuart Sierra's namespace guidelines](https://stuartsierra.com/2015/05/10/clojure-namespace-aliases)
+  unless there is a strong reason not to do so or this style guide says otherwise.
+<sup>[[link](#stuart-sierra-namespace-guideline)]</sup>
+
+* <a name="commonly-used-namespaces"></a>
+  Keep commonly used namespace abbreviations to maintain consistency between services, such as:
```

Comment:
Sierra's namespace guideline itself allows for some deviation (for well estabilished namespaces in the community or very commonly used namespaces). An extensive whitelist might be impossible to maintain, but maybe there is no harm in trying. The ones I listed here were the ones that came up during the discussion with the Bills team, but others could be added as well in a per need basis.